Library/Homebrew/requirements/xcode_requirement.rb

```diff
@@ -22,7 +22,12 @@ def message
       A full installation of Xcode.app#{version} is required to compile this software.
       Installing just the Command Line Tools is not sufficient.
     EOS
-    if MacOS.version >= :lion
+    if Version.new(MacOS::Xcode.latest_version) < Version.new(@version)
```

Comment:
Why is `MacOS::Xcode.latest_version` not already a Version object?

Reply:
We Shall Never Know